### PR TITLE
Add telco modal form

### LINKED ISSUE #8010: Create /telecommunications contact-us modal form
https://docs.google.com/document/d/1GszC6D1EU0_cHXIsOL1HlPtoMuB1XplCK8J2poqyLQk/edit

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Added telco specific contact us form
- **Note: the copy doc wants radio buttons for the first question, but the script doesn't allow it.**

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telecommunications#get-in-touch
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1GszC6D1EU0_cHXIsOL1HlPtoMuB1XplCK8J2poqyLQk/edit#)
- See that the form submits

## Issue / Card

Fixes #8010

### CODE REVIEW

File: templates/shared/forms/interactive/telco.html [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="stage-of-transformation-i-want-to-learn-more-about-nfv">
+            <label for="stage-of-transformation-i-want-to-learn-more-about-nfv">I want to learn more about NFV</label>
+            <input type="checkbox" id="stage-of-transformation-i-am-evaluating-the-need-to-transition-to-nfv">
+            <label for="stage-of-transformation-i-am-evaluating-the-need-to-transition-to-nfv">I am evaluating the need to transition to NFV</label>
+            <input type="checkbox" id="stage-of-transformation-i-am-planning-to-transition-to-nfv-in-the-next-12-months">
+            <label for="stage-of-transformation-i-am-planning-to-transition-to-nfv-in-the-next-12-months">I am planning to transition to NFV in the next 12 months</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="stage-of-transformation-i-have-already-started-the-transition-process,-but-haven’t-implemented-anything-yet">
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/telco.html:25]: These ids are not valid ids. Your need to pull out the commas and `’`

File: templates/shared/forms/interactive/telco.html [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+            <div class="js-other-container">
+              <input type="checkbox" class="js-other-container__checkbox" id="patform-preferences-servers--other">
+              <label for="patform-preferences-servers--other">Other</label>
+              <input type="text" id="patform-preferences-servers--other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other">
+            </div>
+
+          </div>
+          <div class="col-3 u-sv3">
+            <h4>Networking</h4>
+            <input type="checkbox" id="patform-preferences-cisco">
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/telco.html:137]: This is a duplicated ID and I think you meant platform.

File: templates/shared/forms/interactive/telco.html [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+            <label for="patform-preferences-cisco">Cisco</label>
+            <input type="checkbox" id="patform-preferences-juniper">
+            <label for="patform-preferences-juniper">Juniper</label>
+            <input type="checkbox" id="patform-preferences-arista">
+            <label for="patform-preferences-arista">Arista</label>
+            <input type="checkbox" id="patform-preferences-dell">
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/telco.html:143]: This one is duplicated too

File: templates/shared/forms/interactive/telco.html [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+
+            <div class="u-hide">
+              <h3>Your comments</h3>
+              <ul class="p-list">
+                <li class="mktFormReq mktField p-list__item">
+                  <label for="Comments_from_lead__c" class="mktoLabel">What would you like to talk to us about?</label>
+                  <textarea id="Comments_from_lead__c" name="Comments_from_lead__c" rows="5" class="mktoField" maxlength="2000"></textarea>
+                </li>
+              </ul>
+              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formValidation" value="" />
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/telco.html:278]: We probably don't want an aria-label on a hidden element

File: templates/telecommunications/index.html [hunk 5]
```diff
@@ -727,7 +727,7 @@ <h2>
 {% with first_item="_cloud_nfv", second_item="_cloud_newsletter", third_item="_telco_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
 
 <!-- Set default Marketo information for contact form below-->
-<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/general" data-form-id="1257" data-lp-id="2154" data-return-url="https://www.ubuntu.com/contact-us/form/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/telco" data-form-id="1257" data-lp-id="2154" data-return-url="https://www.ubuntu.com/contact-us/form/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
```
Comment (anthonydillon) [inline templates/telecommunications/index.html:730]: Should we add a specific thank you page for the telco section if we are starting to get leads?

Comment (pmahnke) [reply templates/telecommunications/index.html]: yes, but for now Titus is happy with the generic one

**Pull request merged**